### PR TITLE
Add issue and pr templates

### PR DESCRIPTION
**Pull request by chaomengyuan**
address issue #43

### CODE REVIEW

**Comment by Jeffwan** [comment]
I will read the contents and have a check

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Related issue number
+
+<!-- For example: "Closes #1234" -->
+
+## Checks
+
+- [ ] I've made sure the tests are passing. 
+- Testing Strategy
+   - [ ] Unit tests
+   - [ ] Release tests
```
Comment (Jeffwan) [inline .github/pull_request_template.md:18]: We don't have a release test yet.  Let's rename to "Manual tests"

**Comment by Jeffwan** [comment]
Beside the last comment, other looks good to me. Thanks for the contribution

File: .github/pull_request_template.md (hunk 1 above)
Comment (chaomengyuan) [reply .github/pull_request_template.md:18]: Sure. I will change to that.

**Pull request merged**